pkgs/development/python-modules/openshift/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib
+  , buildPythonPackage
+  , fetchPypi
+  , jinja2
+  , kubernetes
+  , ruamel-yaml
+  , six
+  , python-string-utils
+}:
+
+buildPythonPackage rec {
+  pname = "openshift";
+  version = "0.12.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-aggRnD4goiZJPp4cngp8AIrJC/V46378cwUSfq8Xml4=";
+  };
+
+  buildInputs = [
+    jinja2
+    kubernetes
+    python-string-utils
+    ruamel-yaml
+    six
+  ];
+
+  doCheck = false;
```

Comment:
Why? Do the tests not pass? (ref [§15.20.1.2.3](https://nixos.org/manual/nixpkgs/unstable/#testing-python-packages)) Are they not shipped in the Pypi build?

Please leave a comment, at least, and consider using a pythonImportsCheck if disabling tests cannot be avoided.

Reply:
> pythonImportsCheck

Should always be used to make sure check only inputs are not required to import the package.

Reply:
didnt know about pythonImportsCheck, I've wanted that for a long time cool ! thanks